### ISSUE TITLE
This command fails: aragon apm publish major --environment rinkeby --files dist/

### ISSUE THREAD

**Issue by vporton** [aragon/aragon-cli#1502]
## 🐛 Bug Report

```
aragon apm publish major --environment rinkeby --files dist/
```
does not work.

### Have you read the [Contributing Guidelines on issues](https://github.com/aragon/aragon-cli/blob/master/CONTRIBUTING.md#ways-to-contribute)?

- [x] Yes

## To Reproduce

1. Download https://github.com/vporton/courts/tree/b093cb9f82a29c3d729977a3c323e35fd506dbaa
2. Run the above command.

## Expected behavior

Should publish the app.

## Actual Behavior

Produces an error (see below)

#### Mainnet or testnet?

Rinkeby

#### Environment

- OS and OS version: Ubuntu 19.10

```
$ aragon apm publish major --environment rinkeby --files dist/
  ↓ Running prepublish script [skipped]
  ↓ Running prepublish script [skipped]
    → prepublishOnly script not defined in package.json
  ✔ Check IPFS
  ✔ Applying version bump (major)
  ✔ Building frontend
  ✔ Deploy contract
  ✔ Determine contract address for version
  ✔ Prepare files for publishing
  ✔ Generate application artifact
  ✔ Publish intent

 The following information will be published: 
 Contract address: 0x5bdc328f646DcE2AE4A9780B9fa733b7801f1D28 
 Content (ipfs): QmPBCiwSR3auEjog5isz9ed86ZXZAiJw9VwShw4swz3HGo

 Explore the ipfs content locally: 
 http://localhost:8080/ipfs/QmSDgpiHco5yXdyVTfhKxr3aiJ82ynz8V14QcGKicM3rVh/#/explore/QmPBCiwSR3auEjog5isz9ed86ZXZAiJw9VwShw4swz3HGo 

? Publish to reward2.open.aragonpm.eth repo Yes

  ❯ Publish reward2.open.aragonpm.eth
  ❯ Publish reward2.open.aragonpm.eth
    ⠇ Generating transaction
      → Fetching DAO at 0x2DF31C5fcBB94656fc98e2953cF0DA03e2885Cfa...
      Sending transaction
✖ Error inspecting DAO
```

**Comment by vporton** [comment]
Related bug: https://github.com/aragon/aragon-cli/issues/1082

**Comment by litvintech** [comment]
I have the same issue right now. Publishing process to APM in Rinkeby seems broken

**Comment by macor161** [comment]
Hi @litvintech, can you post here the command that you tried?

**Comment by 0xGabi** [comment]
Fixed with #1590
